libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuverView.kt

```diff
@@ -45,17 +47,16 @@ import kotlinx.coroutines.flow.asStateFlow
  */
 class MapboxManeuverView : ConstraintLayout {
 
-    private val _heightFlow = MutableStateFlow(0)
     private val _maneuverViewState = MutableStateFlow<MapboxManeuverViewState>(
         MapboxManeuverViewState.COLLAPSED
     )
+    private val _visibilityFlow = MutableStateFlow(false)
 
     /**
      * Observe on [maneuverViewState] to get notified about changes to [MapboxManeuverViewState].
      */
     val maneuverViewState = _maneuverViewState.asStateFlow()
-
-    internal val heightFlow = _heightFlow.asStateFlow()
+    val visibilityFlow = _visibilityFlow.asStateFlow()
```

Comment:
```suggestion
    internal val visibilityFlow = _visibilityFlow.asStateFlow()
```
But I don't think we should track visibility here. Drop-In uses binders to show or hide views, so it should be enough to simply track whether `ManeuverComponent` is attached at the moment.

Reply:
Agreed. We don't need any changes to `ManeuverView`. Visibility can be directly toggled via `onAttached` and `onDetached` in `ManeuverViewComponent`

Reply:
But you said that in 1TAP you change the visibility of a maneuver view. I found this piece of code: https://github.com/mapbox/1tap-android/blob/master/map/src/main/java/com/mapbox/onetap/map/presentation/ui/NavigationOverlayBinder.kt#L175
With your idea it won't work, will it? We would have no placeholder even though maneuver view was gone.

Reply:
This code is used when Drop-In UI is disabled. When it is enabled, we use [this approach](https://github.com/mapbox/1tap-android/blob/9c379b1dbc28c583f4f2055defcb5d9a9ff761d4/map/src/main/java/com/mapbox/onetap/map/presentation/ui/DropInNavigationOverlayBinder.kt#L158-L160). We set an empty binder in order to hide the view, which also detaches the maneuver component.